gunicorn/arbiter.py

```diff
@@ -509,6 +516,7 @@ def spawn_worker(self):
             util._setproctitle("worker [%s]" % self.proc_name)
             self.log.info("Booting worker with pid: %s", worker_pid)
             self.cfg.post_fork(self, worker)
+            os.environ['INSTANCE'] = str(worker_id)
```

Comment:
Good point - GUNICORN_WORKER_INSTANCE it is.(https://github.com/benoitc/gunicorn/commit/8703fffca4e700445659ca90b24a88faa38bebf0)